charts/tekton-apps/Chart.yaml

```diff
@@ -13,7 +13,7 @@ triggersVersions: "v0.16.0"
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.22
+version: 0.1.23-dev
```

Comment:
I have released the 1.23 already with my changes. so use 1.24-dev pls (and then release AS 1.24) when final

Reply:
done